homeworks/egetoimre/5_oop/car_fleet_mgr.py

```diff
@@ -0,0 +1,68 @@
+class Car:
+    def __init__(self, brand, model, year):
+        self.brand = brand
+        self.model = model
+        self.year = year
+        self.mileage = 0
+        self.fuel_level = 100
+
+    def drive(self, distance):
+        max_distance = self.fuel_level * 10
+        act_distance = min(distance, max_distance)
+
+        if max_distance < distance:
+            print(f"A tüzelőanyag kifogyott, kérem töltse fel a(z) {self.brand} {self.model} autót!")
+
+        self.mileage += act_distance
+        self.fuel_level -= act_distance / 10
+
+    def refuel(self, add):
+        if self.fuel_level + add > 100:
+            print(f"A tüzelőanyagot túltöltötték, a(z) {self.brand} {self.model} autónál! Takarítsa fel!")
+        self.fuel_level = min(self.fuel_level + add, 100)
+
+class Fleet:
+    def __init__(self):
+        self.cars = []
+
+    def add_car(self, car):
+        self.cars.append(car)
+        print(f"A {car.brand} {car.model} hozzáadása megtörtént")
+
+    def remove_car(self, car):
+        self.cars.remove(car)
```

Comment:
itt meg kéne vizsgálni törlés előtt hogy az adott autó tényleg benne van e a flottában:
`if car not in self.cars:` különben ValueError-t kapunk egy nem létező objektum törlésénél.